src/js/hc-scripts/expire-invitations.js

```diff
@@ -0,0 +1,48 @@
+const { createHandler } = require('./utils');
+const { getInvitationExpiryCandidates, expireInvitation } = require('js/server/attendance/logic');
+
+function createExpiryQueue(responsesToProcess, dryRun) {
+  // Defensive clone for mutating array
+  const responses = responsesToProcess.slice(0);
+
+  const expireResponse = (response) => {
+    if (dryRun) {
```

Comment:
Maybe useful to make it clearer from the comments what the 'dryRun' mode does and how to access it